src/dotnet/Chat.UI.Blazor/Components/RecordingSubHeader/RecordingSubHeader.razor

```diff
@@ -64,12 +64,12 @@
         };
 
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
-        var recordingChat = await GetRecordingElsewhereChat(cancellationToken);
-        var stopRecordingAt = await ChatAudioUI.StopRecordingAt.Use(cancellationToken);
+        var recordingChat = await GetRecordingElsewhereChat(cancellationToken).ConfigureAwait(false);
```

Comment:
Check GetRecordingElsewhereChat

Reply:
Its' OK - it uses only injected services and Session